index.js

```diff
@@ -299,10 +299,19 @@ const extractFeatures = async page => {
   }
 }
 
+let sharp = null
+const resizeCanvas = async (canvas, resX, resY) => {
+  if (!sharp) sharp = require("sharp")
+  const sharpCanvas = sharp(canvas)
+  return sharpCanvas.resize(resX, resY).toBuffer()
```

Comment:
Sharp default resize strategy is `cover`. This will cut the image. We want `inside` instead.